src/test.js

```diff
@@ -0,0 +1,42 @@
+nn = {};
+function aa(lim) {
+ if (!nn[lim]) {
+  let num = nn[Object.keys(nn).length - 1] + 1 | 0;
+  let sum = 0;
+  let i = ((Object.keys(nn).length) | 0) - 1;;
+  // let lim = 10;
+  let tempNum = 0;
+  let pow = 0;
+  while (i < lim) {
+   sum = 0;
+   tempNum = num;
+   pow = Math.floor((Math.log10(Math.abs(tempNum)) | 0) + 1);
+   while (tempNum !== 0) {
+    sum += Math.pow(tempNum % 10, pow);
+    tempNum = Math.floor(tempNum / 10);
+   }
+   if (num === sum) {
+    console.log('yes' + sum + ' ' + num);
+    i++;
+    nn[i] = sum;
+   }
+   num = num + 1;
+  }
+ }
+}
+
+
+
+aa(1);
+console.log(nn);
+aa(4);
+console.log(nn);
+aa(10);
+console.log(nn);
+aa(5)
+console.log(nn[10]);
+
+
+
+
+console.log(findValue(nn, 5));
```

Comment:
Good job with the testing!

@prakashchoudhary07 Two suggestions: 

- [ ] Move the tests into a `test` folder, that's a sibling to `src`. We don't want to confuse source code with test code, right? Unnecessary overhead
- [ ] Your tests don't give a good indication of what's working and what's failing. It's just a lot of noisy text. Make it such that you can gauge what the tests are doing easily